package.json

```diff
@@ -19,6 +19,7 @@
   },
   "homepage": "https://github.com/nathan-gs/ha-map-card#readme",
   "dependencies": {
+    "@mdi/js": "^7.4.47",
```

Comment:
You can use mdi icons natively in Home Assistant, using `ha-icon`.

Reply:
Removed the dependency on the @mdi/js package.

Reply:
Sorry for the multiple commits.